resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -128,6 +128,19 @@ Specify the connector in the builder:
 .withProperty(ApiClientConnectorProperty.CONNECTOR_PROVIDER, new OkHttpClientConnectorProvider())
 ```
 
+#### Setting the max retry time
+
+In default, Java SDK makes a call to API without a Retry option but this feature is configurable.
+The SDK configuration is default to Retry off and it can be configured to enable or disable while creating an `ApiClient` Instance.
+
+Specify the amount of time in seconds that client wants to wait while retrying if client opts to enable Retry feature:
+```{"language":"java"}
+.withMaxRetryTimeInSec(30)
+```
+When Retry feature is enabled and if the `status code` form the API response is `429 or 502 or 503 or 504`,
```

Comment:
Remove the rest of this explanation and replace with:

```
When the retry feature is enabled, the SDK adheres to the recommended best practices for automatically retrying retryable responses. The best practices are documented in the [Rate Limiting](https://developer.mypurecloud.com/api/rest/rate_limits.html) Developer Center article.
```

Reply:
done.